net-libs/libwebsockets/libwebsockets-1.7.4.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="canonical libwebsockets.org websocket library"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="static-libs"
+
+DEPEND="
+	dev-util/cmake
+	dev-libs/openssl
+	"
+RDEPEND=""
```

Comment:
openssl is required at runtime too, in addition to zlib. I suggest something like:

```
RDEPEND="
	 dev-libs/openssl:0
	 sys-libs/zlib
"
DEPEND="${RDEPEND}"
```

The ``:0`` on openssl ensures that the slot with headers will be installed (there's another openssl slot that installs only libraries for compatibility with binary-only packages).